src/libNode/FinalBlockProcessing.cpp

```diff
@@ -1518,7 +1572,7 @@ bool Node::ProcessFinalBlock(const vector<unsigned char>& message,
     unsigned int cur_offset = offset;
 
     // Initialize it with 255
```

Comment:
The comment is out of sync with the code. Better remove it or reflect the change.